src/app/api/login/route.ts

```diff
@@ -28,56 +28,50 @@ export const POST = withAppRouterHighlight(async (request: NextRequest) => {
       status: 400,
     });
   }
+  const existingUser = await db
+    .selectFrom("user")
+    .selectAll()
+    .where("email", "=", parsedEmail.data)
+    .executeTakeFirst();
 
-  try {
-    const existingUser = await db
-      .selectFrom("user")
-      .selectAll()
-      .where("email", "=", parsedEmail.data)
-      .executeTakeFirst();
+  if (!existingUser) {
+    return new Response("Wrong password or user does not exist!", {
```

Comment:
Could u extract to a const string such that people don't accidently update one of them and forgot to update the other.